cloudinit/distros/raspberry-pi-os.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (C) 2024-2025 Raspberry Pi Ltd. All rights reserved.
+#
+# Author: Paul Oberosler <paul.oberosler@raspberrypi.com>
+#
+# This file is part of cloud-init. See LICENSE file for license information.
+
+import logging
+
+from cloudinit import subp
+from cloudinit.distros import debian
+
+LOG = logging.getLogger(__name__)
+
+
+class Distro(debian.Distro):
+    @classmethod
+    def get_logger(cls):
```

Comment:
This isn't necessary. See https://docs.pytest.org/en/7.1.x/how-to/logging.html#caplog-fixture or search for other uses of `caplog` in tests.